src/main/kotlin/no/nav/syfo/client/dokarkiv/domain/AvsenderMottaker.kt

```diff
@@ -1,13 +1,13 @@
 package no.nav.syfo.client.dokarkiv.domain
 
 data class AvsenderMottaker private constructor(
-    val id: String,
+    val id: String?,
     val idType: String,
     val navn: String? = null,
 ) {
     companion object {
         fun create(
-            id: String,
+            id: String?,
```

Comment:
Denne kan (i hvert fall teoretisk) bli null for behandler, men api'et skal tåle det (feks tilsvarende i padm2)